benches/signatures.rs

```diff
@@ -39,17 +38,16 @@ fn gen_keypair() -> SigningData {
     let mut msg = [0u8; 32];
     rng.fill_bytes(&mut msg);
     let (k, p) = RistrettoPublicKey::random_keypair(&mut rng);
-    let r = RistrettoSecretKey::random(&mut rng);
     let m = RistrettoSecretKey::from_bytes(&msg).unwrap();
```

Comment:
Likely due to the benches changing over time.

It doesn't affect the timings, since the secret key instantiation is done in the setup step. 

Easy enough to change.